src/test/java/unstable/MilestonePickerTests.java

```diff
@@ -13,21 +13,31 @@
 public class MilestonePickerTests extends UITest {
     @Test
     public void showMilestonePicker() {
-        click("#dummy/dummy_col0_9");
-        press(KeyCode.M);
+        clickIssue(0, 9);
+        push(KeyCode.M);
```

Comment:
I think the `press` function doesn't let go of the key which causes the test to fail (the milestone picker dialog only appears _occasionally_). Also, the typing of `milestones` later will miss out the letter `m`. Had these problems previously which were solved by changing it to `push`.